tools/dev-benchmark-pallet.sh

```diff
@@ -68,28 +68,29 @@ fi
 
 # For each runtime, only update the weights if they already existed before
 # This is because the different runtimes have different pallets
-if [ -f "chains/orchestrator-paras/dancebox/src/weights/$PALLET.rs" ]; then
+# When adding a new pallet, you can set env variable FORCE_COPY=1
```

Comment:
Add information about this in `docs/benchmarking.md`.

Reply:
Done, now I'm thinking that this will break if you add a pallet to some runtimes but not all of them, but that should be easy to fix by modifying the script when it happens...